README.md

```diff
@@ -36,46 +36,47 @@ Symbol type as the safe input type to ActiveRecord query APIs.
 
 The enforcement of the contracted type is done by decorating those query APIs
 that are subject to SQLi, and banning String type input. As some original APIs
-already distinguish Symbol and String type inputs, Symbol is warpped into a new
+already distinguish Symbol and String type inputs, Symbol is wrapped into a new
 trusted type under the safe contract to avoid confusion internally. The trusted
 type can only be constructed from a safe symbol thus achieving
 secure-by-construction.
 
 ## Setup
 
 `SafeActiveRecord.activate!` needs to be called before an application starts to
-process user input, but after earger loading finishes, so that a snapshot of
-safe symbols can be taken. In a rails app, it can done at
+process user input, but after eager loading finishes, so that a snapshot of
+safe symbols can be taken. In a rails app, it can be done at
 [config.after_initialize](https://guides.rubyonrails.org/configuring.html#config-after-initialize).
 
 The method takes in a hash into which a few options can be passed:
 
 *   `safe_query_mode`: `:strict` or `:lax`; `:lax` mode allows usage of
     `RiskilyAssumeTrustedString` type.
 *   `dry_run`: true/false, default to false; when set to true, only warnings
-    will be emitted but otherwise an exception will be raised when uncomforming
+    will be emitted but otherwise an exception will be raised when unconforming
     types are passed in.
 *   `intercept_load`: true/false, default to false; when set to true,
     `require`/`load`/`require_relative` will be intercepted in order to
     calculate the delta of symbols created during new Ruby source code loading,
     and add them to the trusted symbol set. This is to compensate for use cases
-    when eager_load is turned off, typically during local development. See
-    limitation section for the caveats.
+    when eager_load is turned off, typically during local development.
+
+    **WARNING**: make sure `intercept_load` is disabled in all production systems including Rake tasks. See [limitation section](#limitation) for more details.
```

Comment:
Can you run [mdl](https://github.com/markdownlint/markdownlint) against this file to ensure it's consistent?

Reply:
Done thanks!